github_projects_board.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Projects board
```

Comment:
Your questions are very important. 
This is an opportunity to define more accurate process, because right now I think every AM manage it differently.

@efratn @noamoss what is your input to @RachelBaram 's questions?